specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -537,7 +537,7 @@
         "description": "Gets compute definition by its name. Any secrets (storage keys, service credentials, etc) are not returned - use 'keys' nested resource to get them.",
         "x-ms-examples": {
           "Get a AKS Compute": { "$ref": "./examples/getAKSCompute.json" },
-          "Get a AML Compute": { "$ref": "./examples/getAMLCompute.json" }
+          "Get a AML Compute": { "$ref": "./examples/getAmlCompute.json" }

```

Comment:
The example reference name (e.g. `./examples/getAmlCompute.json`) is case sensitive, in this case the actual file name is `getAMLCompute.json` (AML in caps).  This is why the CI is failing and throwing syntax error so you might want to rename the example file to match the reference name.

Reply:
done